contracts/bonding/BondingManager.sol

```diff
@@ -90,34 +94,20 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
         _;
     }
 
-    // Automatically claim token pools shares from lastClaimTokenPoolsSharesRound through the current round
-    modifier autoClaimTokenPoolsShares() {
-        updateDelegatorWithTokenPoolsShares(msg.sender, roundsManager().currentRound());
+    // Automatically claim earnings from lastClaimRound through the current round
+    modifier autoClaimEarnings() {
+        updateDelegatorWithEarnings(msg.sender, roundsManager().currentRound());
         _;
     }
 
-    function BondingManager(address _controller) public Manager(_controller) {}
-
-    /*
-     * @dev Batch set protocol parameters. Only callable by the controller owner
-     * @param _unbondingPeriod Rounds between unbonding and possible withdrawl
-     * @param _numTranscoders Max number of registered transcoders
-     * @param _numActiveTranscoders Number of active transcoders
+    /**
```

Comment:
Curious, can we auto-generate docs from this style of function documentation?

Reply:
Yeah the small updates to the format of these comments were so that they adhere to the [Ethereum Natural Specification](https://github.com/ethereum/wiki/wiki/Ethereum-Natural-Specification-Format) and there are tools for auto-generating docs based upon that format